clients/src/test/java/org/apache/kafka/clients/consumer/internals/HeartbeatRequestManagerTest.java

```diff
@@ -349,6 +408,10 @@ public void testNoCoordinator() {
         when(coordinatorRequestManager.coordinator()).thenReturn(Optional.empty());
         NetworkClientDelegate.PollResult result = heartbeatRequestManager.poll(time.milliseconds());
 
+        when(pollTimer.isExpired()).thenReturn(false);
+        when(pollTimer.remainingMs()).thenReturn(Long.MAX_VALUE);
+        when(heartbeatRequestState.timeToNextHeartbeatMs(time.milliseconds())).thenReturn(1000L);
```

Comment:
needed or wanted for some reason? again wondering because since the state is a spy on the internal component, I would expect to have the real implementation kicking in, based on the interval defined for the HBMgr.

Reply:
These are now removed